cmd/collection/main.go

```diff
@@ -73,6 +73,7 @@ func main() {
 		followerEng       *followereng.Engine
 		colMetrics        module.CollectionMetrics
 		err               error
+		rpcMetricsEnabled bool
```

Comment:
We already have a `Config` struct for the `ingress` component that implements the RPC server (see [here](https://github.com/onflow/flow-go/pull/684/files#diff-4688e4f1e6bc5063d6bcb143912d69e213eff1019cf5caed6f75218349b70bc2R65)). Can we add this new config field there instead?